warehouse/query-core/src/main/java/datawave/query/index/lookup/ScannerStream.java

```diff
@@ -15,7 +15,6 @@
  * Note that certain delayed terms may create a ScannerStream without an underlying RangeStreamScanner.
  */
 public class ScannerStream extends BaseIndexStream {
```

Comment:
This was an artifact of doing dev on Windows and Linux. Will remove this change.